patches/0004-Add-OpenSSL-crypto-backend.patch

```diff
@@ -281,6 +282,10 @@ index 00000000000000..d929b3e516bdb0
 +			}
 +		}
 +	}
++	if openssl.FIPS() {
```

Comment:
The code in the `switch` potentially call `openssl.SetFIPS`, so it can't be directly reused. We could add some logic to workaround that, but this code is already quite tricky and I would prefer not to add more complexity just to save a couple of cgo calls that are only ever called once.